drafts/2019-11-06-1900011.adoc

```diff
@@ -1,13 +1,13 @@
 ---
 layout: post
-title: "190011"
+title: "Printing elapsed time message on completion of slow request with Request Timing 19.0.0.11"
 categories: blog
 author_picture: https://avatars3.githubusercontent.com/tomjenningss
 author_github: https://github.com/tomjenningss
-seo-title: Reactive messaging between microservices with MicroProfile on Open Liberty 19.0.0.11 - OpenLiberty.io
+seo-title: Printing elapsed time message on completion of slow request with Request Timing 19.0.0.11 - OpenLiberty.io
 seo-description: 
 ---
-= Reactive messaging between microservices with MicroProfile on Open Liberty 19.0.0.9
+= Printing elapsed time message on completion of slow request with Request Timing 19.0.0.11
 Tom Jennings <https://github.com/tomjenningss>
 
 
```

Comment:
Insert an intro here; eg
"Open Liberty 19.0.0.11 makes diagnosis of slow or hung requests easier by providing more detailed logging.

If you're interested in what's coming soon in Open Liberty, take a look at our [current development builds](link to section here). In particular, get an early insight into Jaeger support for MicroProfile OpenTracing, and performance improvements through annotation caching."